js/whitebit.js

```diff
@@ -1151,6 +1161,55 @@ module.exports = class whitebit extends Exchange {
         };
     }
 
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        let type = undefined;
+        if (fromAccount === 'main' && toAccount === 'spot') {
+            type = 'deposit';
+        } else if (fromAccount === 'spot' && toAccount === 'main') {
+            type = 'withdraw';
+        }
```

Comment:
Let's support both the unified types (`main`, `spot`) and the exchange-specific types (`main`, `trade`) here:

```JavaScript
        const accountsByType = this.safeValue (this.options, 'accountsByType');
        const fromAccountId = this.safeString (accountsByType, fromAccount, fromAccount);
        const toAccountId = this.safeString (accountsByType, toAccount, toAccount);
        let type = undefined;
        if (fromAccount === 'main' && toAccountId === 'trade') {
            type = 'deposit';
        } else if (fromAccount === 'trade' && toAccount === 'main') {
            type = 'withdraw';
        }
        if (type === undefined) {
            throw new ExchangeError (this.id + ' transfer() only allows transfers between main account and spot (trade) account');
        }
```

Also, let's use uniform exception messages (everywhere):

```JavaScript
throw new BadSymbol (this.id + ' fetchFundingRate() supports contracts only');
throw new ArgumentsRequired (this.id + ' fetchOpenOrders() requires a symbol argument');
...
throw new ExceptionClassName (this.id + ' methodName() blablabla');
```